analyzer/tests/unit/test_option_parser.py

```diff
@@ -352,3 +354,23 @@ def test_ccache_compiler(self):
         action['command'] = 'ccache -Ihello main.cpp'
         res = log_parser.parse_options(action)
         self.assertEqual(res.original_command, 'ccache -Ihello main.cpp')
+
+    def test_compiler_include_file(self):
+        action = {
+            'file': 'main.cpp',
+            'directory': '',
+            'command': 'g++ main.cpp'}
+
+        _, info_file_name = \
+            tempfile.mkstemp(suffix='.json', text=True)
```

Comment:
It would be better to use [`tempfile.NamedTemporaryFile`](https://docs.python.org/2/library/tempfile.html#tempfile.NamedTemporaryFile) in a `with` statement. In that way this file will be automatically deleted at the end of the test.

E.g.:
https://github.com/Ericsson/codechecker/blob/2ebdd29264d1bb5a489cf990d63b88ede8e4ed7c/analyzer/codechecker_analyzer/buildlog/log_parser.py#L421-L423